x/distribution/keeper/keeper.go

```diff
@@ -101,3 +101,14 @@ func (k Keeper) WithdrawValidatorCommission(ctx sdk.Context, valAddr sdk.ValAddr
 
 	return coins, nil
 }
+
+// GetTotalRewards returns the total amount of fee distribution rewards held in the store
+func (k Keeper) GetTotalRewards(ctx sdk.Context) (totalRewards sdk.DecCoins) {
```

Comment:
Does this include commission?

Reply:
It should right?

Reply:
the commission is included on the outstanding rewards